src/components/SearchSploit/SearchSploit.tsx

```diff
@@ -142,122 +122,124 @@ const SearchSploit = () => {
     };
 
+    // Get output type arguments
+    const getOutputTypeArgs = (outputType: string): string[] => {
+        const typeMap: Record<string, string> = {
+            json: "-j",
+            Overflow: "-o",
+            Path: "-p",
+            Verbose: "-v",
+            www: "-w",
+        };
+        return [typeMap[outputType]].filter(Boolean);
+    };
+
+    // Get non-search arguments
+    const getNonSearchArgs = (nonSearch: string): string[] => {
+        const nonSearchMap: Record<string, string> = {
+            Mirror: "-m",
+            Examine: "-x",
+        };
+        return [nonSearchMap[nonSearch]].filter(Boolean);
+    };
+
+    // Clear output
     const clearOutput = useCallback(() => {
         setOutput("");
         setAllowSave(false);
         setHasSaved(false);
-    }, [setOutput]);
+    }, []);
 
     return (
-        <form
-            onSubmit={form.onSubmit((values) =>
-                onSubmit({
-                    ...values,
-                    searchOption: selectedSearchOption,
-                    out_put: selectedout_putOption,
-                    non_search: selectedNonSearchOption,
-                })
-            )}
-        >
-            <LoadingOverlay visible={loading} />
-            {loading && (
-                <div>
-                    <Button variant="outline" color="red" style={{ zIndex: 1001 }} onClick={handleCancel}>
-                        Cancel
-                    </Button>
-                </div>
-            )}
-
-            <Stack>
-                {UserGuide(title, description_userguide)}
-                <TextInput label={"Search Term"} {...form.getInputProps("searchTerm")} />
-                <NativeSelect
-                    value={selectedSearchOption}
-                    onChange={(e) => setSelectedSearchOption(e.target.value)}
-                    label={"Search option"}
-                    data={searchOption}
-                    placeholder={"Pick a Search option"}
-                    description={"Type of Search to perform"}
-                />
-                <NativeSelect
-                    value={selectedout_putOption}
-                    onChange={(e) => setSelectedOut_putOption(e.target.value)}
-                    label={"Output"}
-                    data={out_put}
-                    placeholder={"Select an Output"}
-                    description={"Type of output to display"}
-                />
-                <NativeSelect
-                    value={selectedNonSearchOption}
-                    onChange={(e) => setSelectedNonSearchOption(e.target.value)}
-                    label={"Non-Searching"}
-                    data={non_search}
-                    placeholder={"Select an option"}
-                    description={
-                        "Mirror (aka copies) an exploit to the current working directory, Examine (aka opens) the exploit using $PAGER"
-                    }
+        <>
```

Comment:
You for got to use RederComponent here
```suggestion
         <RenderComponent
            title={title}
            description={description}
            steps={steps}
            tutorial={tutorial}
            sourceLink={sourceLink}
        >
```